website/content/en/preview/AWS/user-data.md

```diff
@@ -46,6 +47,37 @@ spec:
     karpenter.sh/discovery: my-cluster
 ```
 
+This example adds SSH keys to allow remote login to the node (replace *my-authorized_keys* with your key file):
+
+```yaml
+apiVersion: karpenter.k8s.aws/v1alpha1
+kind: AWSNodeTemplate
+metadata:
+  name: bottlerocket-example
+spec:
+  amiFamily: Bottlerocket
```

Comment:
This won't work with Bottlerocket, only AL2 and Ubuntu (at least in userdata directly). Bottlerocket would need the admin container and then this userdata nested in the admin container's user-data from the regular bottlerocket toml user-data config file.

It's probably sufficient just to show it for AL2 though and leave out Bottlerocket.

Reply:
Okay. I fixed the metadata.name and amiFamily. Let me know of other parts of that example need to change.